app/views/describeYourItem.scala.html

```diff
@@ -36,21 +36,32 @@
         @components.error_summary(form.errors)
 
         @components.heading(messages("describeYourItem.heading"))
-
-        <p>@messages("describeYourItem.pullOut.paragraph")</p>
+        @components.subheading(messages("describeYourItem.subheading"))
 
         @components.input_text(
-            field = form("field1"),
-            label = messages("describeYourItem.field1")
+            field = form("name"),
+            label = messages("describeYourItem.name")
         )
 
         @components.input_textarea(
-            field = form("field2"),
-            label = messages("describeYourItem.field2"),
+            field = form("description"),
```

Comment:
Nice one for doing this, just be wary the ids are generated using the field name so a few ATs may be broken

Reply:
We will mop up during the fixing of the TraderUI ATs.   (Or Ash will)